src/patterns/OcAppSideBar.vue

```diff
@@ -41,6 +42,15 @@ export default {
       default: false,
       type: Boolean,
     },
+    /**
+     * Provide the accessible name for the close button, since it has no visible label
+     * but a svg, to which to provide an accessible name would be more complicated.
+     * Value of closeButtonLabel will eventually be the value of the button's aria-label
+     **/
+    closeButtonLabel: {
+      default: "Close",
```

Comment:
This string will need to be translated inside Phoenix.

Reply:
This is considered, see: https://github.com/owncloud/phoenix/blob/bugfix/appmenu-close-btn-label/src/components/Menu.vue#L24